client/package.json

```diff
@@ -90,7 +90,8 @@
     "resize-observer-polyfill": "^1.5.1",
     "serve": "^14.2.1",
     "styled-components": "^6.1.1",
-    "typescript": "5.1.6"
+    "typescript": "5.1.6",
+    "zod": "^3.23.8"
```

Comment:
@atomicgamedeveloper 
Is this used? Overall, it is better to move the config verification logic into `util/config.ts`. The zod can be used there.